src/style.css

```diff
@@ -0,0 +1,123 @@
+.todo-list {
+  display: flex;
+  justify-content: center;
+}
+
+.task-outer-container {
+  border: 3px solid #ecebea;
+  flex-basis: 70vw;
+}
+
+.title-container {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid #ecebea;
+}
+
+.tasks-container-title {
+  font-size: 18px;
+  font-family: 'Helvetica', 'Arial', sans-serif;
+  margin: 0;
+  padding: 20px;
+  user-select: none;
+}
+
+.form-container {
+  /* padding: 15px 20px; */
```

Comment:
Kindly remove this comment, following YAGNI guidelines.

Reply:
Changed successfully, thanks!